hydroshare/local_settings.template

```diff
@@ -1,10 +1,10 @@
-# Copy the contents of this file to a new file local_settings.py
```

Comment:
I'm not clear what the differences between template and cutover are.  Could you explain?

Reply:
Ignoring local_settings.py means that upon checkout, build systems are lacking a local_settings.py .... the legacy build system of chained shell scripts all over the place needs local_settings.py to exist .... in order not to change too many things at once, the local_settings.py is generated from local_settings.cutover to make the shell script beasts happy. In a future effort, someone can refactor the old shell scripts to a more normal build approach. local_settings.template is for any local developer interested in overriding typical settings. Also, until settings.py is reconciled with settings.desired and tested, a local_settings.py file is required.